src/models/batch_auction_model.rs

```diff
@@ -145,7 +146,7 @@ pub struct FeeModel {
 pub struct InteractionData {
     pub target: H160,
     pub value: U256,
-    #[derivative(Debug(format_with = "debug_bytes"))]
```

Comment:
This is still worth having with a small fix as explained [here](https://github.com/cowprotocol/services/pull/460)

Reply:
Nice suggestion. thanks. I will do it in the next PR